staging/src/k8s.io/apiserver/pkg/server/signal.go

```diff
@@ -44,6 +45,26 @@ func SetupSignalHandler() <-chan struct{} {
 	return stop
 }
 
+// SetupSignalContext is same as SetupSignalHandler, but a context.Context is returned.
+// Only one of SetupSignalContext and SetupSignalHandler should be called, and only can
+// be called once.
```

Comment:
put the same note on SetupSignalHandler docs

Reply:
done